docs/DeveloperGuide.md

```diff
@@ -278,8 +278,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | Teaching Assistant                                        | delete a specific student from TAPA                                                    | remove student who has withdrawn from the module during the semester                          |
 | `* * *`  | Teaching Assistant                                        | list students alphabetically                                                           | print students in a list as their names would appear on the official class list               |
-| `* * *`  | Teaching Assistant                                        | save existing data to hard drive                                                       | access the data again when I reopen the app                                                   |
+| `* * *`  | Teaching Assistant                                        | tag students                                                                           | know what modules they are taking                                                             |
+| `* * *`  | Teaching Assistant                                        | filter students by their tags                                                          | view the progress of students in each separate tag (different class/module)                   |
+| `* * *`  | Teaching Assistant                                        | define my own tasks for each module                                                    | customise the tasks to each module                                                            |
+| `* * *`  | Teaching Assistant                                        | assign tasks to multiple students                                                      | allocate tasks to students without having to individually assign them                         |
+| `* * *`  | Teaching Assistant                                        | undo the most recent command entered                                                   | revert any unwanted changes (such as deleting student) without much trouble                   |
+| `* * *`  | Teaching Assistant                                        | delete a task given to a student                                                       | delete the task from the list of tasks                                                        |
+| `* * *`  | Teaching Assistant                                        | list the students who have/have not completed a specific task                          | track the completion status of a specific task                                                |
 | `* *`    | Teaching Assistant                                        | see all the tasks that a student currently have                                        | observe his/her progress                                                                      |
```

Comment:
```suggestion
| `* *`    | Teaching Assistant                                        | see all the tasks that a student currently has                                        | observe his/her progress                                                                      |
```

Reply:
Ok I will be fixing the grammar, thanks for pointing this out